CORE_DEV_GUIDE.md

```diff
@@ -165,8 +165,13 @@ community values. After nomination voting will happen on a private mailing list.
 While it is expected that most votes will be unanimous, a two-thirds majority of
 the cast votes is enough.
 
-Core developers can choose to become emeritus core developers and suspend
-their approval and voting rights until they become active again.
+## Offboarding Core Members
+
+Core developers are expected to make regular contributions to the project. Core developers
+can choose to become emeritus core developers and suspend their approval and voting rights
+until they become active again. If a core developer becomes inactive in the project for a
+period of one year, they may be removed by core developer team with a two-thirds majority
```

Comment:
```suggestion
period of one year, they may be removed by core developer team with a majority
```

I think we want to bias towards making it easy to remove inactive core devs, in order to avoid getting back to the same situation. 2/3 is a high threshold. What if many people are inactive? It might be impossible to achieve a quorum.

Reply:
(I read this to mean 2/3 of those voting)

Reply:
I agree this is ambiguous. I chose two-thirds to match the threshold required to elect a new member. It seemed to me that these should match. In general, I like a lower threshold for both actions. The current situation in Zarr-Python shows how hard it is to get even a majority vote accomplished 🤦 !